app/queries/get_application_choices_for_providers.rb

```diff
@@ -20,6 +20,6 @@ def self.call(providers:)
       .where('courses.provider_id' => providers, 'courses.recruitment_cycle_year' => RecruitmentCycle.visible_years)
       .or(ApplicationChoice.includes(*includes)
         .where('courses.accredited_provider_id' => providers, 'courses.recruitment_cycle_year' => RecruitmentCycle.visible_years))
-      .where('status IN (?)', ApplicationStateChange.states_visible_to_provider).includes([:accredited_provider])
+      .where('status IN (?)', ApplicationStateChange.states_visible_to_provider_in_vendor_api).includes([:accredited_provider])
```

Comment:
Will this not also hide these applications in Manage? https://github.com/DFE-Digital/apply-for-teacher-training/blob/ceeb842702cd47f3ae94b49cb9194f422c8cf060/app/controllers/provider_interface/application_choices_controller.rb#L12-L14

Reply:
ah, yes! I will. Thanks!